helm/hpcc/templates/_helpers.tpl

```diff
@@ -2527,3 +2527,16 @@ spec:
   {{- end -}}
  {{- end -}}
 {{- end -}}
+
+{{/*
+A template to generate component annotations, merges statid efault annotatiosn with global and component annotations
```

Comment:
typos....

Reply:
an impressive number of typos on 1 line!